src/sentry/eventstream/kafka/backend.py

```diff
@@ -69,44 +70,29 @@ def encode_bool(value: Optional[bool]) -> str:
                 value = False
             return str(int(value))
 
+        def encode_dict(value: Mapping[Any, Any]) -> str:
+            return json.dumps(value)
+
         # we strip `None` values here so later in the pipeline they can be
         # cleanly encoded without nullability checks
         def strip_none_values(value: Mapping[str, Optional[str]]) -> Mapping[str, str]:
             return {key: value for key, value in value.items() if value is not None}
 
-        # transaction_forwarder header is not sent if option "eventstream:kafka-headers"
-        # is not set to avoid increasing consumer lag on shared events topic.
-        transaction_forwarder = self._is_transaction_event(event)
-
-        send_new_headers = options.get("eventstream:kafka-headers")
```

Comment:
Is this safe to delete? I looked for references and the only place this value is defined is in `src/sentry/options/defaults.py` and it just defaults to True. In what scenario would this not be True?

Reply:
Based on what @lynnagara said I'm not totally sure if we should delete it without some more discussion. I'm in favour of deleting these paths, but others might disagree long term.

The option should be safe to delete. It is also set in the `sentry_options` table. I double checked and the value is `True` there as well

Reply:
Please don't delete this yet even though it may not be used. It was not supposed to be the default. We had to flip it because we were having big problems scaling single tenant.